src/components/Problem/ProblemCard.vue

```diff
@@ -27,7 +27,7 @@ function downloadTestCase(problemId: number) {
       <div v-if="error" class="alert alert-error shadow-lg">
         <div>
           <i-uil-times-circle />
-          <span>Oops! Something went wrong when loading announcements.</span>
+          <span>Oops! Something went wrong when loading problem.</span>
```

Comment:
Is this only for problem loading error handling?

Reply:
Yes, the `error` would only be `truthy` when exception raised during problem fetching